app/js/controllers/foyer.js

```diff
@@ -84,9 +84,11 @@ angular.module('ddsApp').controller('FoyerCtrl', function($scope, $state, $state
     });
 
     $scope.$on('rfr', function(e, rfr) {
-        $scope.situation.rfr = rfr;
-        $scope.situation.ressourcesYearMoins2Captured = true;
-        $scope.$broadcast('ressourcesYearMoins2Captured');
+        if (rfr !== undefined) {
+            $scope.situation.rfr = rfr;
+            $scope.situation.ressourcesYearMoins2Captured = true;
```

Comment:
This value is never reverted. What happens if the user goes back to the page and erases the fields?